dashboard/test/ui/features/stage_lock.feature

```diff
@@ -36,7 +36,8 @@ Scenario: Lock settings for students
   When I sign out
   And I sign in as "Teacher_bobby"
   And I am on "http://studio.code.org/s/allthethings"
-  And I select the first section
```

Comment:
this step is no longer necessary, because we now default to the teacher's most recently-created section. it also creates timing problems, since it causes the page to unload/reload.